### PR TITLE
More detailed hardware wallet instructions

### PR DESCRIPTION
**Pull request by rbkhmrcr**
This PR adds more detailed instructions for getting the coda Ledger app up and running on the hardware wallet.

### CODE REVIEW

File: frontend/website/pages/hardware-wallet.mdx [hunk 1]
```diff
@@ -19,9 +19,81 @@ The following debian packages are needed to use hardware wallets on linux:
...
 
+### Make sure your computer is recognising the Ledger device
+
+- Plug in the Ledger device, unlock it with your passcode, and go to the menu. Don't open any app,
+- Run `lsusb`, and see if the device is listed -- it will have ID `2c97:0001`
+- If the Ledger device is not listed, follow the suggestions [here](https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues).
+
+### Build/download the Coda Ledger application
+
+Either download the precompiled app binary from [here](...).
```
Comment (imeckler) [inline frontend/website/pages/hardware-wallet.mdx:30]: Link?

Comment (rbkhmrcr) [reply frontend/website/pages/hardware-wallet.mdx:30]: i didn't know where to put it, in the coda-ledger-app repo too?

File: frontend/website/pages/hardware-wallet.mdx [hunk 2]
```diff
@@ -19,9 +19,81 @@ The following debian packages are needed to use hardware wallets on linux:
...
+
+#### System-wide installation
```
Comment (bkase) [inline frontend/website/pages/hardware-wallet.mdx:32]: Can we move the Build-from-source instructions to the bottom of the docs -- that way people that want to download a precompiled binary won't have to scroll past the build instructions before getting to the `Using the wallet` section.

File: frontend/website/pages/hardware-wallet.mdx [hunk 3]
```diff
@@ -12,20 +13,67 @@ Note: macOS and Windows are not officially supported at this time.
...
+added to `/etc/udev/rules.d/` (e.g. with `sudo vim /etc/udev/rules.d/20-hw1.rules`),
+changing `<UNIX username>` to your unix username:
+
+```
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0000", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0001", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="2c97", ATTRS{idProduct}=="0004", MODE="0660", TAG+="uaccess", TAG+="udev-acl" OWNER="<UNIX username>"
+```
+
+To load the precompiled app onto the Ledger device, run the following:
```
Comment (Schmavery) [inline frontend/website/pages/hardware-wallet.mdx:56]: Are these scripts in git@github.com:CodaProtocol/ledger-coda-app.git?
We might need to move the ` ### Download the Coda Ledger app` step to here?

File: frontend/website/pages/hardware-wallet.mdx (hunk 1 above)
Comment (Schmavery) [reply frontend/website/pages/hardware-wallet.mdx]: Yeah maybe in the github releases section of that repo?

File: frontend/website/pages/hardware-wallet.mdx (hunk 3 above)
Comment (rbkhmrcr) [reply frontend/website/pages/hardware-wallet.mdx:56]: they'll be in the link in the 'Build/download the Coda Ledger application' above, but I'll add that for context here too

**Pull request merged**